lib/game.rb

```diff
@@ -1,69 +1,22 @@
 module TicTacToe
 
-  class Game
-
-    def initialize
-      @board = Board.new
-      @eval_game = EvalGame.new
-      @console_int = ConsoleInterface.new(WrapperIO.new)
-      @player_won = nil
-    end
-
-    def run
-      symbol_selection
-      game_loop  
-      finished_game
-    end
-    
-  private 
-
-    def symbol_selection
-      @console_int.select_symbol
-      @console_int.puts_space
-      @console_int.puts_symbols
-      @console_int.puts_space
-    end
-
-    def game_loop
-      until @eval_game.game_over?(@board.nine_space_array)
-        player_move(1, @console_int.player1_symbol) 
-        unless @eval_game.game_over?(@board.nine_space_array)
-          player_move(2, @console_int.player2_symbol) 
-        else
-          @player_won = 1
-        end
-      end
-     @player_won = 2 unless @player_won 
-    end
+  class Game 
     
-    def valid_move_loop(move, symbol)
-      until @board.fill(move, symbol)
-        @console_int.valid_move_message
-        move = @console_int.move_messages(@board.nine_space_array)
-      end
-      move
+    def initialize(game_turn, marker_loop, end_game)
+      @game_turn = game_turn
+      @marker_loop = marker_loop
+      @end_game = end_game
+      @tied_or_player_won = nil
     end
 
-    def player_move(num, symbol)
-      @console_int.player_turn_message(num)
-      move = @console_int.move_messages(@board.nine_space_array)
-      move = valid_move_loop(move, symbol)
-      @board.fill(move, symbol)
-    end
-
-    def tied_or_won_message
-      if @eval_game.won?(@board.nine_space_array)
-        @console_int.won_message(@player_won)
-      else
-        @console_int.tied_message
-        @console_int.puts_space
+    def run 
+      marker1, marker2 =  @marker_loop.marker_selection
+      until @tied_or_player_won
+        @tied_or_player_won = @game_turn.execute(1, marker1)
```

Comment:
If you use the EvalGame's #game_over? to check the status, you will lose the logic that determines the game result in this class. To me, that seems like something that would be within the scope of EvalGame's responsibilities.